frontend/src/components/authentication/CreateAccountForm.jsx

```diff
@@ -0,0 +1,328 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import validateName, { validateEmail, validatePassword } from "../../helpers/regexValidator";
+import "../../css/create-account-form.css";
+
+const styles = {
+  createAccountContent: {
+    padding: "12px 32px 0 32px"
+  },
+  inputTitle: {
+    padding: "12px 0 6px 0",
+    fontWeight: "bold"
+  },
+  inputs: {
+    width: "100%",
+    padding: "3px 6px 3px 6px",
+    borderRadius: 4
+  },
+  inputForNames: {
+    width: 190,
+    padding: "3px 6px 3px 6px",
+    borderRadius: 4
+  },
+  iconForNames: {
+    color: "#278400",
+    position: "absolute",
+    margin: "8px 0 0 170px"
+  },
+  iconForOtherFields: {
+    color: "#278400",
+    position: "absolute",
+    margin: "8px 0 0 370px"
+  },
+  loginBtn: {
+    width: 150,
+    display: "block",
+    margin: "24px auto"
+  },
+  validationError: {
+    color: "red",
+    marginTop: 6
+  }
+};
+
+let SUBMIT_BTN_DISABLED;
```

Comment:
I remember seeing a comment about why this can't go in state, but I can't seem to find it anymore. It is important to keep in state, so if I can help debug that issue let me know.

Reply:
Refactored in commit #https://github.com/code-for-canada/project-thundercat/commit/e5846956d1b656df900c4cf147563d004a7baabe